core/IO.carp

```diff
@@ -18,6 +18,8 @@ module are wrappers around the C standard library.")
     (register get-char (Fn [] Int) "getchar") 
     (doc      fgetc "gets a character from file (thin wrapper for the from C standard library).")
     (register fgetc (Fn [(Ptr FILE)] Int) "fgetc") 
+    (doc fputc "writes a character to a file (thin wrapper for the from C standard library).")
```

Comment:
"for the from" seems to be missing a word?

Reply:
good catch, fixed!